app/models/subnet/ipv6.rb

```diff
@@ -26,6 +26,10 @@ def self.supported_ipam_modes
     [:eui64, :db, :none]
   end
 
+  def self.supports_tftp?
+    false
```

Comment:
@dLobatog : Just a short answer for now, don't have a lot of time today:

For tftp orchestration to work, we need to make changes to `app/models/concerns/orchestration/tftp.rb` before it can work for ipv6. There should be "subnet6" somewhere, haven't given it much thought right now. My main problem is that it's actually IPv4 and IPv6 independent. So when a host has an IPv4 subnet and a IPv6 subnet and both have a tftp proxy set, which one do you choose. That's why I wanted to hide it in the UI for now. We can easily add it later again when it's properly tested.

To provision a server with ipv6 only you would need the bios to make a dhcpv6 request. Most bioses don't support this. Most EFI firmwares do though. That's why I wrote we need EFI support. For VMWare you have to tell the compute resource to do an EFI boot and explicity set it to boot via IPv6. fog-vsphere doesn't support this right now. So we'd (read: I'd) have to add this first.

Does that make sense to you? If you want, we can do a hangout and discuss things there.